lib/chef/knife/ec_restore.rb

```diff
@@ -308,6 +308,8 @@ def restore_group(chef_fs_config, group_name, includes = {:users => true, :clien
         end
 
         group.write(members.to_json)
+      rescue Chef::ChefFS::FileSystem::NotFoundError => e
```

Comment:
This is a fairly large function, do we know what call we expect to fail with this when the group doesn't exist?  Might be worth a comment.